### PR TITLE
auth_keycloak: improve scripts

### PR DESCRIPTION
**Pull request by simahawk**
Store default values on each run so to not type them over and over
for calls to the same service.

### CODE REVIEW

**Review by yvaucher — APPROVED** [review]

File: auth_keycloak/examples/get_token.py [hunk 1]
```diff
@@ -2,49 +2,80 @@
...
+    'username': UID,
+    'password': PWD,
+    'client_id': CLIENT_ID,
+    'client_secret': CLIENT_SECRET,
+}
+
+if os.path.isfile(DEFAULTS_DATA_FILE):
+    click.echo('Loading defaults from %s' % DEFAULTS_DATA_FILE)
+    with open(DEFAULTS_DATA_FILE, 'r') as ff:
+        default_conf = json.loads(ff.read())
```
Comment (tarteo) [inline auth_keycloak/examples/get_token.py:34]: It would be nice to overwrite only the values that exist in the json file.

**Review by tarteo — APPROVED** [review]
LGTM

File: auth_keycloak/examples/get_token.py (hunk 1 above)
Comment (tarteo) [reply auth_keycloak/examples/get_token.py:34]: Or check if all the keys exist in the json file.

Comment (hbrunn) [reply auth_keycloak/examples/get_token.py:34]: is this comment a show stopper or should i merge?

Comment (tarteo) [reply auth_keycloak/examples/get_token.py:34]: It's not a show stopper, you can merge.

**Comment by simahawk** [comment]
@dreispt I think there's an issue w/ travis conf. To be fixed. I check later.

**Comment by hbrunn** [comment]
I hade the same problem with a v10 PR that pulls server-tools via its requirements. And the only solution I came up with is https://github.com/OCA/project/pull/654/commits/0400673eb7169be8f3a9f3ba85c46c7ef3ed3d8b

**Comment by simahawk** [comment]
fixed upstream, rebased

**Pull request merged**